sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormField.java

```diff
@@ -34,24 +34,24 @@
     /*
      * The text value field..
      */
-    private final FieldText valueText;
+    private final FieldData valueData;
 
     /**
      * Constructs a FormField object.
      *
      * @param confidence The confidence of the recognized field.
-     * @param labelText The label text value for the field.
+     * @param labelData The label text value for the field.
```

Comment:
docstring should be updated.
I think we say:
"Contains the text, bounding box, and field elements for the field label."